packages/reports/src/Components/market-symbol-icon-row.tsx

```diff
@@ -1,10 +1,26 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { extractInfoFromShortcode, isHighLow } from '@deriv/shared';
 import { Icon, Popover, IconTradeTypes } from '@deriv/components';
 import { getMarketName, getTradeTypeName } from '../Helpers/market-underlying';
 
-const MarketSymbolIconRow = ({ icon, payload, show_description, should_show_multiplier = true }) => {
+type TMarketSymbolIconRow = {
+    action: string;
```

Comment:
there is no action props in the `MarketSymbolIconRow`, you can remove it from here